ansible/roles/etcd/tasks/main.yml

```diff
@@ -11,4 +11,4 @@
 
 - name: Call restart etcd2
   include: etcd2_restart.yml
-  when: etcd2_modified == true and etcd_started.changed == false
+  when: etcd2_modified == true and etcd2_started.changed == false
```

Comment:
@stephenrlouie ok, let's keep it that way and refactor it later if we find enough reasons to do that.